extension/devtools/views/SelectorEdit.html

```diff
@@ -172,13 +184,22 @@
 
     <!-- insertValue -->
     <div class="form-group feature feature-insertValue">
-        <label for="insertValue" class="col-sm-2 control-label">Text value</label>
+        <label for="insertValue" class="col-sm-2 control-label">Input value</label>
 
         <div class="col-sm-8">
             <input type="text" class="form-control" name="insertValue" id="insertValue" placeholder="Text value to insert" value="{{selector.insertValue}}">
         </div>
     </div>
 
+    <!-- ConstantValue -->
+    <div class="form-group feature feature-constantValue">
+        <label for="constantValue" class="col-sm-2 control-label">Input constant</label>
+
+        <div class="col-sm-8">
+            <input type="text" class="form-control" name="constantValue" id="constantValue" placeholder="Text constant to insert" value="{{selector.constantValue}}">
+        </div>
+    </div>
+
```

Comment:
I suggest we can use the same feature insertValue for both InputValue and ConstantValue.